File: Numbers/GPSeries.java
public class GPSeries {
    public static double GPcheck(double a,double r,int n){
        double first=a,commonRatio=r;
        int NumOfTerms=n;
        double sum=0;
        for(int i=1;i<=n;i++){
        sum+=first;
        first=first*commonRatio;

    }
    return sum;
}
public static void main(String args[]){
    double a=12.5;
    double r=10.5;
    int n=5;
        System.out.println("The sum of the GP series:"+ GPcheck(a, r, n));
    }
    
}

File: Problem Solving Python/Numbers/GP_Series.py
def GPSeries(a,r,n):
    first_term=a
    common_ratio=r
    no_terms=n
    sum=0
    for i in range(1,n+1):
        sum += first_term
        first_term *= common_ratio
    return sum

if __name__=='__main__':
    a=float(input("Enter the first term of the GP series: "))
    r=float(input("Enter the common ratio of the GP series: "))
    n=int(input("Enter the number of terms of GP series: "))
    print("The sum of the GP series is: ",GPSeries(a,r,n))